ks-nav/kern_bin_db/CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing to kern_bin_db
+
+:+1::tada: First off, thanks for taking the time to 
+contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to kern_bin_db.
+The kern_bin_db utility is hosted in the 
+[kern_bin_db repository](https://github.com/alessandrocarminati/kern_bin_db-) 
```

Comment:
Should this refer to the ELISA GitHub repo instead?

Reply:
Same as above.

Reply:
Same as above

Reply:
This has been solved in pull request 12